### ISSUE TITLE
l2 regularization?

### ISSUE THREAD

**Issue by ZimaBlue307** [mindspore-ai/mindspore#202]
L1 Regularizer is described in [here](https://www.mindspore.cn/docs/en/r1.7/api_python/nn/mindspore.nn.L1Regularizer.html?highlight=l1regularizer). But what about applying l2 regularization to weights? something similar is described in here: [ActivityRegularization](https://keras.io/api/layers/regularization_layers/activity_regularization/)

**Comment by cxl-ll** [comment]
Hello, the question has been received. We will reply after confirmation.

**Comment by MindWilliam** [comment]
currently L2 regularization is implemented by weight decay in optimizer . So mindspore is not support the interface of L2 Regularization.  If you want to use it, you can using the operation of mindspore to combine it.
